### PR TITLE
Feature: redesign create project form

### LINKED ISSUE #250: Redesign create project from template form
## Description

The purpose of this task is to redesign the form "create project from template"

Current design:
image

Expected design
image
image

### PR DESCRIPTION
**Pull request by Zorin95670**
Resolved #250

### CODE REVIEW

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/card/PluginsCard.vue [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<template>
+  <q-card>
+    <q-card-section>
+      <div class="text-h6">
+        Plugins
```
Comment (justinehell) [inline src/components/card/PluginsCard.vue:5]: needs i18n

File: src/components/form/CreateProjectTemplateForm.vue [hunk 2]
```diff
@@ -1,28 +1,51 @@
...
-        v => isUniqueProjectName($t, projectNames, v)
-      ]"
-      filled
-      lazy-rules
-      data-cy="name-input"
-    />
+    <div class="row q-gutter-md">
+      <div class="flex-2">
+        <div class="text-subtitle2">
+          Rename this template for your project
```
Comment (justinehell) [inline src/components/form/CreateProjectTemplateForm.vue:10]: needs i18n

File: src/components/form/CreateProjectTemplateForm.vue [hunk 3]
```diff
@@ -1,28 +1,51 @@
...
+          :rules="[
+            v => notEmpty($t, v),
+            v => isUniqueProjectName($t, projectNames, v)
+          ]"
+          outlined
+          lazy-rules
+          data-cy="name-input"
+        />
+        <div class="text-subtitle2 q-mb-md">
+          Content
```
Comment (justinehell) [inline src/components/form/CreateProjectTemplateForm.vue:24]: needs i18n

File: src/components/form/CreateProjectTemplateForm.vue [hunk 4]
```diff
@@ -1,28 +1,51 @@
...
+        </div>
+        <q-input
+          v-model="templateDescription"
+          filled
+          data-cy="content-input"
+          disable
```
Comment (justinehell) [inline src/components/form/CreateProjectTemplateForm.vue:30]: no linter error ? data-cy must be last (if no methods/events)

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/composables/TemplateManager.js [hunk 5]
```diff
@@ -61,6 +61,7 @@ export async function getTemplatesByType(type, pluginName = '') {
       ...definition,
       isTemplate: true,
       plugin: template.plugin,
+      plugins: template.plugins,
```
Comment (justinehell) [inline src/composables/TemplateManager.js:64]: Update tests ?

**Review by justinehell — CHANGES_REQUESTED** [review]

File: src/components/form/CreateProjectTemplateForm.vue [hunk 6]
```diff
@@ -184,4 +207,10 @@ watch(() => props.isChecked, () => {
 .create-project-template-form {
   min-width: 400px;
 }
+.flex-1 {
+  flex: 1;
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue:211]: no, it's not a default value. So we need to add it.

File: src/components/form/CreateProjectTemplateForm.vue [hunk 7]
```diff
@@ -47,10 +47,19 @@
         lazy-rules
         data-cy="token-input"
       />
-    </template>
-    <div class="flex row items-center justify-center">
+    </div>
+    <div class="flex row items-center justify-between q-mt-md">
```
[the remark this answers is not in the document]
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue:74]: done

File: src/components/form/CreateProjectTemplateForm.vue (hunk 2 above)
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue:10]: done

File: src/components/form/CreateProjectTemplateForm.vue (hunk 3 above)
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue:24]: done

File: src/components/form/CreateProjectTemplateForm.vue (hunk 4 above)
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue:30]: done ad no linter error

File: src/components/card/PluginsCard.vue (hunk 1 above)
Comment (Zorin95670) [reply src/components/card/PluginsCard.vue:5]: done

File: src/composables/TemplateManager.js (hunk 5 above)
Comment (Zorin95670) [reply src/composables/TemplateManager.js:64]: done

**Review by justinehell — APPROVED** [review]

File: src/components/form/CreateProjectTemplateForm.vue [hunk 8]
```diff
@@ -82,9 +113,9 @@ import {
 } from 'src/composables/Project';
 import { getTemplateFileByPath } from 'src/composables/TemplateManager';
 import { FileInput } from 'leto-modelizer-plugin-core';
+import PluginsCard from 'components/card/PluginsCard.vue';
```
Comment (k-y0u) [inline src/components/form/CreateProjectTemplateForm.vue:116]: src/components/card/PluginsCard.vue

**Review by k-y0u — CHANGES_REQUESTED** [review]

File: src/components/form/CreateProjectTemplateForm.vue (hunk 8 above)
Comment (Zorin95670) [reply src/components/form/CreateProjectTemplateForm.vue]: done

**Review by justinehell — APPROVED** [review]

**Pull request merged**